meta-resin-fido/recipes-core/systemd/systemd_%.bbappend

```diff
@@ -1,8 +1,8 @@
 FILESEXTRAPATHS_append := ":${THISDIR}/${PN}"
 
 SRC_URI_append = " \
-	file://0001-Don-t-run-specific-services-in-container.patch
-	file://0002-remove_systemd-getty-generator.patch
+	file://0001-Don-t-run-specific-services-in-container.patch \
```

Comment:
Does this commit need to be squashed in somewhere?

Reply:
It could but it touches a series of patches so I might leave it like this.